lib/actions-oclif/device/init.ts

```diff
@@ -116,7 +117,8 @@ export default class DeviceInitCmd extends Command {
 		const tmpPath = (await tmpNameAsync()) as string;
 		try {
 			logger.logDebug(`Downloading OS image...`);
-			await this.downloadOsImage(tmpPath, application.device_type, options);
+			const osVersion = options['os-version'] || 'default';
```

Comment:
This defaulting to `'default'` here prevents the printing of the warning message regarding a default version being used.

Reply:
Yeah, that was the previous behaviour, so I kept it.  Would you prefer it changed?

Reply:
Just realised that the help for `os-version` in `balena help device init` is quite detailed:
```
		'os-version': flags.string({
			description: stripIndent`
				exact version number, or a valid semver range,
				or 'latest' (includes pre-releases),
				or 'default' (excludes pre-releases if at least one stable version is available),
				or 'recommended' (excludes pre-releases, will fail if only pre-release versions are available),
				or 'menu' (will show the interactive menu)
				`,
		}),
```
And a user might type `--os-version 'default'`, in which case being warned about a default version being used (if that's what they just typed) is not particularly helpful.  Nonetheless, yes, I think it makes sense to remove ` || 'default'` on this line.